noir-projects/aztec-nr/aztec/src/oracle/random.nr

```diff
@@ -6,5 +6,21 @@ pub unconstrained fn random() -> Field {
     rand_oracle()
 }
 
+/// Returns as many random bytes as specified through N.
+pub unconstrained fn get_random_bytes<let N: u32>() -> [u8; N] {
```

Comment:
Moved this function here as I feel like it's a better place for it than the bytes-fields conversion utils.